src/test/java/TicketServiceTest.java

```diff
@@ -0,0 +1,65 @@
+import com.example.models.Ticket;
+import com.example.repository.TicketRepository;
+import com.example.services.TicketService;
+import models.TicketType;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import java.time.LocalDateTime;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+public class TicketServiceTest {
+
+    @Mock
+    private TicketRepository ticketRepository;
+
+    @InjectMocks
+    private TicketService ticketService;
+
+    @BeforeEach
+    public void setUp() {
+        MockitoAnnotations.openMocks(this);
+    }
+
+    @Test
+    public void testMaxWeightAccordingEventCode() {
+        assertEquals(0.000f, ticketService.maxWeightAccordingEventCode((short) 0));
+        assertEquals(3.500f, ticketService.maxWeightAccordingEventCode((short) 100));
+        assertEquals(4.200f, ticketService.maxWeightAccordingEventCode((short) 500));
+        assertEquals(5.300f, ticketService.maxWeightAccordingEventCode((short) 700));
```

Comment:
for such cases you can use parametrized tests https://www.baeldung.com/parameterized-tests-junit-5

Reply:
thanks for recomend